src/transformers/models/opt/modeling_flax_opt.py

```diff
@@ -245,7 +244,7 @@ def __call__(
             attention_bias = lax.select(
                 attention_mask > 0,
                 jnp.full(attention_mask.shape, 0.0).astype(self.dtype),
-                jnp.full(attention_mask.shape, float("-inf")).astype(self.dtype),
+                jnp.full(attention_mask.shape, -1e9).astype(self.dtype),
```

Comment:
@patrickvonplaten I've noticed that encoder-decoder models in Flax use `float("-inf")` here. While this is typically not a problem, as most generation cases don't set the decoder attention mask, we might run into the same problem if the user tries setting it. Shall we also make the `float("-inf")` -> `-1e9` change in those models?

Reply:
I think we should ! I remember other models suffering from the same issue!

Reply:
Good with me - we can update Bart and preserve the `# Copied from ...` statement

Reply:
@sanchit-gandhi done ;) (this updated a few models with the same issue)

Reply:
Amazing, ty @gante 🙌